spec.bs

```diff
@@ -1340,6 +1343,49 @@ Note: This is because we need to ensure that we do not leak <var ignore>creator<
 document's referrer|referrer=], [=Document/origin=], [=creator base url=], [=Document/policy
 container=], across the fenced frame boundary.
 
+<h3 id=local-policy-container-monkeypatch>Local Requests Policy Container Patch</h3>
+
+When making local requests, <{iframe}>s inherit their [=Document/policy container=] from their
+[=navigable container=]'s [=Node/node document=]. If <{fencedframe}>s were to do the same thing,
+that would allow information about the initiator's [=IP address space=] to leak across a fenced
```

Comment:
Certainly it would be the entire policy container that would leak, right? Not just the address space.